### ISSUE TITLE
Compilation fails with Zephyr 2.4

### ISSUE THREAD

**Issue by ufechner7** [bdkrae/zephyr_teensy4_test#1]
When I use the latest stable build of Zephyr compiling this test with:
`ufechner@builder:~/repos/zephyr_teensy4_test$ west build -p -b teensy40 .`

fails:

```
-- Configuring done
-- Generating done
-- Build files have been written to: /home/ufechner/repos/zephyr_teensy4_test/build
-- west build: building application
[1/185] Preparing syscall dependency handling

[48/185] Building C object zephyr/CMakeFiles/zephyr.dir/subsys/disk/disk_access_usdhc.c.obj
/home/ufechner/zephyrproject/zephyr/subsys/disk/disk_access_usdhc.c:2795:12: warning: 'disk_usdhc_init' defined but not used [-Wunused-function]
 2795 | static int disk_usdhc_init(const struct device *dev)
      |            ^~~~~~~~~~~~~~~
[178/185] Building C object zephyr/CMakeFiles/zephyr.dir/drivers/ethernet/eth_mcux.c.obj
FAILED: zephyr/CMakeFiles/zephyr.dir/drivers/ethernet/eth_mcux.c.obj 
ccache /home/ufechner/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/arm-zephyr-eabi-gcc -DBOARD_FLASH_SIZE="CONFIG_FLASH_SIZE*1024" -DBUILD_VERSION=zephyr-v2.4.0-3-gc2a0b0f50bdb -DCPU_MIMXRT1062DVL6A -DKERNEL -DNDEBUG -DXIP_BOOT_HEADER_ENABLE=1 -D_FORTIFY_SOURCE=2 -D__PROGRAM_START -D__ZEPHYR__=1 -I/home/ufechner/zephyrproject/zephyr/include -Izephyr/include/generated -I/home/ufechner/zephyrproject/zephyr/soc/arm/nxp_imx/rt -I/home/ufechner/zephyrproject/zephyr/subsys/usb -I/home/ufechner/zephyrproject/zephyr/subsys/net/l2 -I/home/ufechner/zephyrproject/modules/hal/cmsis/CMSIS/Core/Include -I/home/ufechner/zephyrproject/modules/hal/nxp/mcux/devices/MIMXRT1062 -I/home/ufechner/zephyrproject/modules/hal/nxp/mcux/components/phyksz8081 -I/home/ufechner/zephyrproject/modules/hal/nxp/mcux/drivers/imx/. -I/home/ufechner/zephyrproject/modules/hal/nxp/mcux/middleware/usb/./include -I/home/ufechner/zephyrproject/modules/hal/nxp/mcux/middleware/usb/device/. -I/home/ufechner/zephyrproject/modules/hal/nxp/mcux/middleware/usb/device/./port/zephyr -I/home/ufechner/zephyrproject/modules/hal/nxp/mcux/middleware/usb/phy/. -isystem /home/ufechner/zephyrproject/zephyr/lib/libc/minimal/include -isystem /home/ufechner/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/include -isystem /home/ufechner/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/include-fixed -Os -imacros /home/ufechner/repos/zephyr_teensy4_test/build/zephyr/include/generated/autoconf.h -ffreestanding -fno-common -g -mcpu=cortex-m7 -mthumb -mabi=aapcs -imacros /home/ufechner/zephyrproject/zephyr/include/toolchain/zephyr_stdint.h -Wall -Wformat -Wformat-security -Wno-format-zero-length -Wno-main -Wno-pointer-sign -Wpointer-arith -Wno-address-of-packed-member -Wno-unused-but-set-variable -Werror=implicit-int -fno-asynchronous-unwind-tables -fno-pie -fno-pic -fno-strict-overflow -fno-reorder-functions -fno-defer-pop -fmacro-prefix-map=/home/ufechner/zephyrproject/zephyr=ZEPHYR_BASE -fmacro-prefix-map=/home/ufechner/zephyrproject=WEST_TOPDIR -ffunction-sections -fdata-sections -std=c99 -nostdinc -MD -MT zephyr/CMakeFiles/zephyr.dir/drivers/ethernet/eth_mcux.c.obj -MF zephyr/CMakeFiles/zephyr.dir/drivers/ethernet/eth_mcux.c.obj.d -o zephyr/CMakeFiles/zephyr.dir/drivers/ethernet/eth_mcux.c.obj -c /home/ufechner/zephyrproject/zephyr/drivers/ethernet/eth_mcux.c
/home/ufechner/zephyrproject/zephyr/drivers/ethernet/eth_mcux.c: In function 'eth_tx':
/home/ufechner/zephyrproject/zephyr/drivers/ethernet/eth_mcux.c:744:11: error: too few arguments to function 'ENET_SendFrame'

```

Any idea?

**Comment by ufechner7** [comment]
Commenting out the line:
CONFIG_NETWORKING=y
in prj.conf fixed the issue. :smile:

**Comment by ufechner7** [comment]
Fixed by #2 .

**Comment by ufechner7** [comment]
Works now.
